docs/user/lectures.rst

```diff
@@ -0,0 +1,164 @@
+.. _lectures:
+
+Lectures
+========
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+Overview
+--------
+Apart from :ref:`Exercises` and :ref:`Exams`, Artemis can also host the **Lectures** of a course.
+
+Students
+^^^^^^^^
+
+Students can see all published lectures of a course, grouped by date, by clicking on the tab labeled *Lectures*.
+
+|lectures-overview|
+
+Clicking on a lecture leads to following page, where students can see the title, description, lecture units, and attachments.
+
+|lecture-details|
+
+Instructors
+^^^^^^^^^^^
+Instructors can upload lecture slides, divide lectures into units, integrate video streams, lecture recordings, and exercises into lectures, and define learning goals.
+On the course management site, clicking on |lectures-management-btn| opens the following page for managing lectures.
+
+|lectures-management|
+
+Lectures
```

Comment:
Maybe "Creating Lectures" or "Managing Lectures"? This section is only about how instructors can work with them.